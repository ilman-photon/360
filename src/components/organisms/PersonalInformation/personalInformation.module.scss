.issuedCardContainer {
  border: 1px dashed #6c6c6c;
  max-height: 183px;
  overflow: hidden;
  display: flex;
  align-items: center;
  justify-content: center;

  img {
    margin: 4px;
  }
}
