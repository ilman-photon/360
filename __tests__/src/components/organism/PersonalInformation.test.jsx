import { render, fireEvent, waitFor } from "@testing-library/react";
import "@testing-library/jest-dom";
import PersonalInformation from "../../../../src/components/organisms/PersonalInformation/personalInformation";

window.scrollTo = jest.fn();

describe("PersonalInformation Components", () => {
  let container;
    const mockUserdata = {
        address: "645 Benedict Cliff",
age: 63,
city: "Daphneeshire",
dob: "2022-08-18T13:08:18.012Z",
email: "Justus4@gmail.com",
firstName: "Karlie",
gender: "Transgender",
issuedCardBack: "https://loremflickr.com/275/173",
issuedCardFront: "https://loremflickr.com/275/173",
lastName: "Ernser",
mobile: "(706) 509-6731",
name: "Rupert Jerde",
preferredCommunication: "email",
preferredName: "---",
profilePhoto: "https://loremflickr.com/640/480",
ssn: 3777306119,
state: "South Dakota",
title: "Mrs.",
zip: "03245",
    };

  beforeEach(async () => {
    const mockCallBack = jest.fn();
    container = render(<PersonalInformation isEditing={false} userData={mockUserdata} 
      OnEditClicked={mockCallBack}
      OnCancelEditClicked={mockCallBack}
      OnSaveClicked={mockCallBack}/>)

  await waitFor(() => container.getByText("Preferred Name"));
  });


  it("PersonalInformation render", () => {
    expect(container).toMatchSnapshot();
  });

  it("PersonalInformation View render", () => {

    expect(container.getByText("Name")).toBeInTheDocument();
    expect(container.getByText("Preferred Name")).toBeInTheDocument();
    expect(container.getByText("Title")).toBeInTheDocument();
    expect(container.getByText("Date of Birth")).toBeInTheDocument();
    expect(container.getByText("Age")).toBeInTheDocument();
    expect(container.getByText("Gender")).toBeInTheDocument();
    expect(container.getByText("SSN")).toBeInTheDocument();

    expect(container.getByText("Rupert Jerde")).toBeInTheDocument();
    expect(container.getByText("---")).toBeInTheDocument();
    expect(container.getByText("Mrs.")).toBeInTheDocument();
    expect(container.getByText("8/18/2022")).toBeInTheDocument();
    expect(container.getByText(63)).toBeInTheDocument();
    expect(container.getByText("Transgender")).toBeInTheDocument();
    expect(container.getByText("***-***-6119")).toBeInTheDocument();
    expect(container.getByText("Please upload a photo of government-issued ID, such as Driverâ€™s License or State-issued ID.")).toBeInTheDocument();
  });

  test("is edit button clicked", async () => {

    container.rerender(<PersonalInformation isEditing={true} userData={mockUserdata} />);

        const field1 = container.getByLabelText("Name")
        expect(field1).toBeDisabled();

    const field2 = container.getByLabelText("Preferred Name")
    expect(field2.value).toEqual("---");
    fireEvent.change(field2, { target: { value: "test field 2" } });
    expect(field2.value).toEqual("test field 2");
    
      expect(container.getByText("Month, date, year")).toBeInTheDocument();  

      await waitFor(() => container.getByText("Date of Birth"));
      await waitFor(() => container.getByText("Title"));

    const field3 = container.getByTestId("styled-select-title")
    expect(field3).toBeTruthy();

    const field4 = container.getByLabelText("Age")
    expect(field4.value).toEqual("63");
    expect(field4).toBeDisabled();

    const field5 = container.getByTestId("styled-select-gender")
    expect(field5).toBeTruthy();

    const field6 = container.getByLabelText("SSN")
    expect(field6.value).toEqual("3777306119");
    expect(field1).toBeDisabled();

    expect(container.getAllByRole("button", { name: "Change File" })[0]).toBeVisible();
});


test("is cancel button clicked", async () => {
  container.rerender(<PersonalInformation isEditing={true} userData={mockUserdata} />);

        const field = container.getByLabelText("Age")
        expect(field.value).toEqual("63");
        fireEvent.change(field, { target: { value: 45 } });
        expect(field.value).toEqual("45");
    const cancelButton = container.getByRole("button", { name: "Cancel" });
    fireEvent.click(cancelButton);

    expect(field.value).toEqual("63");
});

test("is save button clicked", () => {
  container.rerender(<PersonalInformation isEditing={true} userData={mockUserdata} />);

        const field = container.getByLabelText("Age")
        expect(field.value).toEqual("63");
        fireEvent.change(field, { target: { value: 45 } });
        expect(field.value).toEqual("45");

    const saveButton = container.getByRole("button", { name: "Save" });
    fireEvent.click(saveButton);


    expect(field.value).toEqual("45");
});

})