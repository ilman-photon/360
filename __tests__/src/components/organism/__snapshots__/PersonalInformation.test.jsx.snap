// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalInformation Components PersonalInformation render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
      >
        <div
          class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
        >
          <div
            class="css-1lwbda4-MuiStack-root"
            style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Photo
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  <img
                    height="122"
                    src="https://loremflickr.com/640/480"
                    style="border-radius: 50%;"
                    width="122"
                  />
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Name
                </div>
                <svg
                  aria-hidden="true"
                  aria-label="If you wish to change this information, please contact Customer Support"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                  data-mui-internal-clone-element="true"
                  data-testid="ErrorOutlineOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  Rupert Jerde
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Preferred Name
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  ---
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Title
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  Mrs.
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Date of Birth
                </div>
                <svg
                  aria-hidden="true"
                  aria-label="If you wish to change this information, please contact Customer Support"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                  data-mui-internal-clone-element="true"
                  data-testid="ErrorOutlineOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  8/18/2022
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Age
                </div>
                <svg
                  aria-hidden="true"
                  aria-label="If you wish to change this information, please contact Customer Support"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                  data-mui-internal-clone-element="true"
                  data-testid="ErrorOutlineOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  63
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  Gender
                </div>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  Transgender
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="labelContainer"
              >
                <div
                  class="label"
                >
                  SSN
                </div>
                <svg
                  aria-hidden="true"
                  aria-label="If you wish to change this information, please contact Customer Support"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                  data-mui-internal-clone-element="true"
                  data-testid="ErrorOutlineOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                  />
                </svg>
              </div>
              <div
                class="css-1okj3ks-MuiStack-root"
              >
                <div
                  style="margin-top: 10px;"
                >
                  ***-***-6119
                </div>
              </div>
            </div>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
            />
            <div>
              <h3
                class="MuiTypography-root MuiTypography-h3 css-1mie9p3-MuiTypography-root"
              >
                State Issued ID
              </h3>
              <div
                class="MuiTypography-root MuiTypography-bodyRegular css-15atyh3-MuiTypography-root"
              >
                Please upload a photo of government-issued ID, such as Driverâ€™s License or State-issued ID.
              </div>
              <div
                class="css-14w0e8k-MuiStack-root"
              >
                <div
                  class="css-1p5q5e5-MuiStack-root"
                >
                  <div
                    class="labelContainer"
                  >
                    <div
                      class="label"
                    >
                      Front Card
                    </div>
                  </div>
                  <div
                    class="css-1okj3ks-MuiStack-root"
                  >
                    <div
                      style="margin-top: 10px;"
                    >
                      <div
                        class="issuedCardContainer"
                      >
                        <img
                          alt="front-id"
                          height="175"
                          src="https://loremflickr.com/275/173"
                          width="267"
                        />
                      </div>
                    </div>
                    <span
                      class="MuiTypography-root MuiTypography-bodySmallRegular css-1r97m9f-MuiTypography-root"
                    >
                      JPG or PNG file formats only. (File size limit is 4 MB)
                    </span>
                  </div>
                </div>
                <div
                  class="css-1p5q5e5-MuiStack-root"
                >
                  <div
                    class="labelContainer"
                  >
                    <div
                      class="label"
                    >
                      Back Card
                    </div>
                  </div>
                  <div
                    class="css-1okj3ks-MuiStack-root"
                  >
                    <div
                      style="margin-top: 10px;"
                    >
                      <div
                        class="issuedCardContainer"
                      >
                        <img
                          alt="back-id"
                          height="175"
                          src="https://loremflickr.com/275/173"
                          width="267"
                        />
                      </div>
                    </div>
                    <span
                      class="MuiTypography-root MuiTypography-bodySmallRegular css-1r97m9f-MuiTypography-root"
                    >
                      JPG or PNG file formats only. (File size limit is 4 MB)
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        style="position: fixed; bottom: 16px; right: 16px;"
      >
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall  css-2ak3pm-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Edit
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
      >
        <div
          class="css-1lwbda4-MuiStack-root"
          style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Photo
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                <img
                  height="122"
                  src="https://loremflickr.com/640/480"
                  style="border-radius: 50%;"
                  width="122"
                />
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Name
              </div>
              <svg
                aria-hidden="true"
                aria-label="If you wish to change this information, please contact Customer Support"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                data-mui-internal-clone-element="true"
                data-testid="ErrorOutlineOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                Rupert Jerde
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Preferred Name
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                ---
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Title
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                Mrs.
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Date of Birth
              </div>
              <svg
                aria-hidden="true"
                aria-label="If you wish to change this information, please contact Customer Support"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                data-mui-internal-clone-element="true"
                data-testid="ErrorOutlineOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                8/18/2022
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Age
              </div>
              <svg
                aria-hidden="true"
                aria-label="If you wish to change this information, please contact Customer Support"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                data-mui-internal-clone-element="true"
                data-testid="ErrorOutlineOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                63
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                Gender
              </div>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                Transgender
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="labelContainer"
            >
              <div
                class="label"
              >
                SSN
              </div>
              <svg
                aria-hidden="true"
                aria-label="If you wish to change this information, please contact Customer Support"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1yagxpu-MuiSvgIcon-root"
                data-mui-internal-clone-element="true"
                data-testid="ErrorOutlineOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11 15h2v2h-2v-2zm0-8h2v6h-2V7zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
            </div>
            <div
              class="css-1okj3ks-MuiStack-root"
            >
              <div
                style="margin-top: 10px;"
              >
                ***-***-6119
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <div>
            <h3
              class="MuiTypography-root MuiTypography-h3 css-1mie9p3-MuiTypography-root"
            >
              State Issued ID
            </h3>
            <div
              class="MuiTypography-root MuiTypography-bodyRegular css-15atyh3-MuiTypography-root"
            >
              Please upload a photo of government-issued ID, such as Driverâ€™s License or State-issued ID.
            </div>
            <div
              class="css-14w0e8k-MuiStack-root"
            >
              <div
                class="css-1p5q5e5-MuiStack-root"
              >
                <div
                  class="labelContainer"
                >
                  <div
                    class="label"
                  >
                    Front Card
                  </div>
                </div>
                <div
                  class="css-1okj3ks-MuiStack-root"
                >
                  <div
                    style="margin-top: 10px;"
                  >
                    <div
                      class="issuedCardContainer"
                    >
                      <img
                        alt="front-id"
                        height="175"
                        src="https://loremflickr.com/275/173"
                        width="267"
                      />
                    </div>
                  </div>
                  <span
                    class="MuiTypography-root MuiTypography-bodySmallRegular css-1r97m9f-MuiTypography-root"
                  >
                    JPG or PNG file formats only. (File size limit is 4 MB)
                  </span>
                </div>
              </div>
              <div
                class="css-1p5q5e5-MuiStack-root"
              >
                <div
                  class="labelContainer"
                >
                  <div
                    class="label"
                  >
                    Back Card
                  </div>
                </div>
                <div
                  class="css-1okj3ks-MuiStack-root"
                >
                  <div
                    style="margin-top: 10px;"
                  >
                    <div
                      class="issuedCardContainer"
                    >
                      <img
                        alt="back-id"
                        height="175"
                        src="https://loremflickr.com/275/173"
                        width="267"
                      />
                    </div>
                  </div>
                  <span
                    class="MuiTypography-root MuiTypography-bodySmallRegular css-1r97m9f-MuiTypography-root"
                  >
                    JPG or PNG file formats only. (File size limit is 4 MB)
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      style="position: fixed; bottom: 16px; right: 16px;"
    >
      <button
        class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root customButton customButtonSmall  css-2ak3pm-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Edit
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
